### PR TITLE
Record the VPN process crashes as GLOBAL

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/414730916066338/1202000697555267/f

### Description
The crashes coming from the `vpn` process do not currently appear in SAFE, but now that it's more mainstream we do want to have them in there. See [this task](https://app.asana.com/0/414730916066338/1202000697555267/f) for more context.

### Steps to test this PR
Just code review, we're basically copy-pasting the logic from the `AlertingUncaughtExceptionHandler`

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #1818** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/1818" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: vpn/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnUncaughtExceptionHandler.kt [hunk 1]
```diff
@@ -53,6 +61,22 @@ class VpnUncaughtExceptionHandler(
...
+            try {
+                uncaughtExceptionRepository.recordUncaughtException(originalException, UncaughtExceptionSource.GLOBAL)
+                offlinePixelCountDataStore.applicationCrashCount += 1
+            } catch (e: Throwable) {
+                Timber.e(e, "Failed to record exception")
+            } finally {
+                originalHandler?.uncaughtException(thread, originalException)
+            }
+        }
+    }
```
Comment (joshliebe) [inline vpn/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnUncaughtExceptionHandler.kt:79]: Should probably add a test for this, can copy the one from `AlertingUncaughtExceptionHandler`.

**Review by joshliebe — APPROVED** [review]
Just the missing test, otherwise LGTM! 🚀

File: vpn/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnUncaughtExceptionHandler.kt (hunk 1 above)
Comment (aitorvs) [reply vpn/src/main/java/com/duckduckgo/mobile/android/vpn/service/VpnUncaughtExceptionHandler.kt:79]: Makes sense, Will add the tests

**Pull request merged**